tools/cldr-apps/js/src/esm/cldrProgress.mjs

```diff
@@ -122,11 +122,7 @@ function insertWidget(spanId) {
     console.error(
       "Error loading ProgressMeters vue " + e.message + " / " + e.name
     );
-    cldrNotify.error(
-      `${e.name} while loading ProgressMeters.vue`,
-      e.message,
-      cldrNotify.NO_TIMEOUT
-    );
+    cldrNotify.exception(e, "while loading ProgressMeters.vue");
```

Comment:
Same. I wonder if we can refactor some functions around the mount() calls.   But in any event, this is great progress.

Reply:
Yes, I'll do some refactoring around mount() and also prefix the exception() message with "Internal error"... who knows what e.name would even be if mount() ever throws an exception; I'll try to force that to happen in testing, to see what it looks like

Reply:
@srl295 Here's how cldr.exception looks with added "Internal error":

![image](https://github.com/unicode-org/cldr/assets/24991506/ea1f0f5f-8027-4546-90e1-636cf33dc7c8)

OK?